data/infra/resources/apt_package.yaml

```diff
@@ -60,6 +61,14 @@ actions_list:
   :unlock:
     markdown: Unlocks the apt package so that it can be upgraded to a newer version.
 properties_list:
+- property: anchor_package_regex
+  ruby_type: true, false
+  required: false
+  default_value: 'false'
+  new_in: '18.3'
+  description_list:
+  - markdown: A Boolean flag to indicate whether packages should have begin and end
+      anchors added to the regex pattern
```

Comment:
```suggestion
  - markdown: A Boolean flag that allows (`false`) or prevents (`true`) apt_package 
      from matching the named package with packages by regular expression if it can't find a package with the exact same name. 
```

Reply:
done